app/Filament/Resources/ServerCredentialResource.php

```diff
@@ -49,9 +49,10 @@ public static function form(Form $form): Form
 
         return $form
             ->schema([
-                Section::make('Server Credential')
-                    ->icon(self::ICON)
-                    ->schema($schema),
+                Section::make('Server Credentials')
+                ->icon('tabler-id-badge-2')
+                ->schema($schema)
+                ->columns(2)
```

Comment:
i don't think we should be changing this. this is still just a single credential.